README.md

```diff
@@ -44,11 +44,11 @@ Example XML config:
 
 ### Configuration parameters
 
-- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. Optional, by default false, meaning the gRPC exporter will be used.
-- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. 
-If left empty, the default values will be used (optional)
-- **Headers** : Used to set OtlpExporterOptions.Headers (optional)
-- **ServiceName** : Used to set the service.name resource (optional)
+- **Endpoint** : Determines where the exporter should send logs. Used to set OtlpExporterOptions.Endpoint. When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_ENDPOINT`
+- **UseHttp** : Determines whether logs are exported using gRPC or HTTP. (optional). When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_PROTOCOL`
+- **Headers** : Used to set OtlpExporterOptions.Headers (optional). When not specified it will use environment-variable: `OTEL_EXPORTER_OTLP_HEADERS`
+- **ServiceName** : Used to set the service.name resource (optional). When not specified it will use environment-variable: `OTEL_SERVICE_NAME`
+- **UseDefaultResources** : Exclude default ressource like: telemetry.sdk.name, telemetry.sdk.language and telemetry.sdk.version. When not specified it will use environment-variable: `OTEL_RESOURCE_ATTRIBUTES`
```

Comment:
This isn't quite true. I'd rewrite it like this:
Use default resources like: telemetry.sdk.name, telemetry.sdk.language and telemetry.sdk.version. By default true.
If set to true, resources will be determined by the environment variable OTEL_RESOURCE_ATTRIBUTES. Setting any resources manually wil override the environment variable.